includes/GbWeiss.php

```diff
@@ -9,6 +9,8 @@
 
 namespace GbWeiss\includes;
 
+use Towa\GebruederWeissSDK\{Configuration, Api\ReadApi};
```

Comment:
I would not include the use statement here since the classes are not used at the moment.